src/cljs/rems/administration/blacklist.cljs

```diff
@@ -26,25 +27,100 @@
              :error-handler (flash-message/default-error-handler :top description)}))
    {:db (assoc db ::loading? true)}))
 
+(rf/reg-event-fx
+ ::add-to-blacklist
+ (fn [{:keys [db]} [_ resource user comment]]
+   (let [description [text :t.administration/add]]
+     (post! "/api/blacklist/add"
+            {:params {:blacklist/resource (select-keys resource [:resource/ext-id])
+                      :blacklist/user (select-keys user [:userid])
```

Comment:
good that `user` and `resource` are maps with standard keys ... but I wonder if there's a way to document this